pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -628,3 +631,20 @@ def parse_basic_auth(auth: Optional[Dict[str, str]]) -> BasicAuth:
             username=auth.get("username"),
             password=auth.get("password"),
         )
+
+    @staticmethod
+    def create_middleware_pipeline() -> MiddlewarePipeline:
+        """Create middleware pipeline.
```

Comment:
If you apply the changes as discussed below, this function will shrink to one line and one log message, and I think you don't need a wrapper function for that. Just include it in the controller method itself.